distro/configs/openmrs/initializer_config/privileges/privileges_core-demo.csv

```diff
@@ -34,3 +34,4 @@ dfc8b5f0-c736-4bc0-8f23-427f0d9c7e7f,Manage Cohort Definitions,Add/Edit/Remove C
 0a12b76e-2175-4eb7-af0b-498129ea1cac,Run Reports,Schedule the running of a report,
 a98c5b27-8d44-42b6-9b84-626cbcc5a227,Manage OrderTemplates,Base privilege for add/edit/delete order templates,
 ccb5ec3a-8f8c-4652-8c60-78f9344c59fb,MSF Administrator,Gives MSF administrative access,
+d0bb8de7-8076-4a83-adbe-76db099466b7,Manage Sensitive Form Fields,Gives MSF user access to edit sensitive form fields,
```

Comment:
This is specific to mental health forms so we should make the privilege specific for example `Edit sensitive mental health fields`